Automattic/wp-calypso
# Set help center search query based on selected block

Related to pdm0RZ-L3-p2

## Proposed Changes

Updated `useCurrentPageSearchMapping` hook to use editor selected block to return relevant Help Center search results.


## Testing Instructions



* Checkout branch.
* Sandbox public API and test site.
* Navigate to `cd apps/editing-toolkit` and run `yarn dev --sync`.
* Load Calypso, open the help center, and navigate to a few different routes to verify the route-to-query mapping is still working
* Navigate to the site editor, open the Help Center, add a block (such as a paragraph block), and verify the selected block changes the search results to return relevant resources.

## Pre-merge Checklist
